packages/EasyErrorHandler/docs/laravel/lumen_install.md

```diff
@@ -33,4 +33,18 @@ $app = new Laravel\Lumen\Application(\dirname(__DIR__));
 $app->configure('easy-error-handler');
 ```
 
+### Change Exception Handler
```

Comment:
That's not really true 😄 
We are already binding the package exception handler to the illuminate contract within `EasyErrorHandlerServiceProvider.php` so no need to do it here again.
What the documentation should say is to make sure the package works as expected they remove any binding of the exception handler contract.